spring/README.md

```diff
@@ -0,0 +1,93 @@
+Cucumber Spring
+===============
+
+Use Cucumber Spring to manage state between steps and for scenarios.
+
+Add the `cucumber-spring` dependency to your pom.xml:
+
+```xml
+<dependencies>
+  [...]
+    <dependency>
+        <groupId>io.cucumber</groupId>
+        <artifactId>cucumber-spring</artifactId>
+        <version>${cucumber.version}</version>
+        <scope>test</scope>
+    </dependency>
+  [...]
+</dependencies>
+```
+
+## Annotation Based Configuration
+
+For your own classes:
+
+* Add the `@Component` annotation to each of the classes cucumber-spring should manage.
+
+* Add a `@Scope("cucumber-glue")` annotation to have cucumber-spring remove them **after each scenario**
```

Comment:
This strikes me as counter intuitive but it is how the code is working.

Reply:
Does this need still to be updated? I'm open to improvements, but figured *some* docs are better than *no* docs ;) (unless blatantly incorrect)

Reply:
That is why merged already. If you want you can add it.